integration/tls_to_backends_test.go

```diff
@@ -104,18 +109,12 @@ var _ = Describe("TLS to backends", func() {
 
 			resp, err := http.ReadResponse(x.Reader, &http.Request{})
 			Expect(err).NotTo(HaveOccurred())
-			resp.Body.Close()
-
 			Expect(resp.StatusCode).To(Equal(404))
 
-			// client-side conn should have been closed
-			// we verify this by trying to read from it, and checking that
-			//  - the read does not block
-			//  - the read returns no data
-			//  - the read returns an error EOF
-			n, err := conn.Read(make([]byte, 100))
-			Expect(n).To(Equal(0))
-			Expect(err).To(Equal(io.EOF))
+			data, err := io.ReadAll(resp.Body)
```

Comment:
The value of `err` here is never checked. Can you either check it, or assign to `_`?

Reply:
Done.